tfhe/src/high_level_api/mod.rs

```diff
@@ -16,6 +16,12 @@ macro_rules! expand_pub_use_fhe_type(
                     [<Compact $fhe_type_name>],
                     [<Compact $fhe_type_name List>],
                     [<$fhe_type_name Id>],
+
+                    // ConformanceParams
+                    [<$fhe_type_name ConformanceParams>],
+                    [<Compressed $fhe_type_name ConformanceParams>],
+                    [<Compact $fhe_type_name ConformanceParams>],
+                    [<Compact $fhe_type_name ListConformanceParams>],
```

Comment:
Macro never called on bool types it seems

Reply:
Yes because its not meant for FheBool

Reply:
Its not compatioble with FheBool because it export the `Id` type which the FheBool do not have